tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -40,7 +40,7 @@ limitations under the License.
 #include "tensorflow/stream_executor/stream.h"
 #include "tensorflow/stream_executor/stream_executor_pimpl.h"
 // clang-format off
-#include "cuda/include/cudnn.h"
+#include "cuda/cudnn.h"
```

Comment:
Can we revert the changes to this file entirely, since we're not expecting any alterations?